geotrek/api/v2/serializers.py

```diff
@@ -488,7 +488,7 @@ def get_description_teaser(self, obj):
             return get_translation_or_dict('description_teaser', self, obj)
 
     class TouristicContentSerializer(TouristicModelSerializer):
-        attachments = AttachmentSerializer(many=True)
+        attachments = AttachmentSerializer(many=True, source='sorted_attachments')
```

Comment:
Shouldn't we also add it on Infrastructure, SensitiveArea ?

Reply:
The attribute `sorted_attachments` is available with the `PicturesMixin`.

The `PicturesMixin` is not associated with the following objects (therefore we cannot do the same modification) : 
- Signage
- Infrastructure
- SensitiveArea